api/app/signals/apps/api/v1/urls.py

```diff
@@ -128,6 +128,14 @@
     )
 )
 
+signal_router_v1.urls.append(
+    path(
+        'public/category/predict',
+        v1_public_views.MLPredictCategoryView.as_view({'get': 'retrieve'}),
```

Comment:
Current ml tool expects a POST, so this is not a straight equivalent. Also we have to think about the REST vs. RPC style calls.

Reply:
See v0 proxy that we agreed on